<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QObject {
background-color: blue;
border: 0px solid;
margin: 0px;
padding: 0px;
border-color: rgb(0, 0, 255);
}
QWidget {
color: black;
font: 16px &quot;Segoe UI&quot;;
}</string>
  </property>
  <widget class="QWidget" name="mb_1">
   <widget class="QWidget" name="mb_voca_adj" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>165</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QObject {
margin: 0px;
background-color: transparent;
}</string>
    </property>
    <widget class="QWidget" name="mb_voca_btn_adj" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>15</width>
       <height>650</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
margin: 0px;
font: 10px;
}
QPushButton#mb_voca_expand {
padding-left: 0px;
padding-right: 0px;
padding-top: 20px;
padding-bottom: 20px;
border-top-right-radius: 5px;
border-bottom-right-radius: 5px;
background-color: rgb(230, 230, 233);
font: 10px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="mb_voca_spcacer_top">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mb_voca_expand">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>◀</string>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="mb_voca_spcacer_bottom">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="mb_voca_main" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>150</width>
       <height>700</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
padding: 5px;
background-color: rgb(230, 230, 233);
}
QWidget.QLabel {
color: rgb(30, 30, 30);
font: 18px;
font-weight: bold;
qproperty-alignment: AlignCenter;
}
QListWidget::item {
background-color: rgb(255, 255, 255); 
border-radius: 5px;
padding: 5px;
padding-left: 10px;
padding-right: 10px;
color: black;
}
QListWidget::item:hover {
background-color: rgb(0, 155, 255); 
color: white;
font-weight: bold;
}
QWidget.QPushButton#mb_voca_open {
font: 16px;
color: white;
border-radius: 5px;
background-color: rgb(0, 155, 255);
}
QWidget.QPushButton#mb_voca_open:hover {
font: 16px;
color: white;
border-radius: 5px;
background-color: rgba(0, 155, 255, 80);
}
QListView {margin: 0px;}
QListView::item {
margin:4px;
}</string>
     </property>
     <widget class="QLabel" name="mb_voca_title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>81</y>
        <width>151</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Word Wrap</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="mb_voca_word">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>119</y>
        <width>151</width>
        <height>581</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>jump</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>see</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fly</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="mb_voca_open">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>New file</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mb_show_adj" native="true">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>0</y>
      <width>821</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
background-color: white;
}
QPushButton#back, 
QPushButton#forward {
background-color: rgb(230, 230, 230);
border-radius: 10px;
}

QLabel#mb_show_eng {
font: 100px;
font-weight: bold;
}
QLabel#mb_show_kor {
font: 50px;
font-weight: bold;
}</string>
    </property>
    <widget class="QWidget" name="mb_show_btns_adj" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>630</y>
       <width>821</width>
       <height>57</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="mb_show_spacer_left">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="mb_show_btn_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="back">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>◀</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dummy">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forward">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>▶</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="mb_show_spacer_right">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="mb_show_eng">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>821</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>jump</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mb_show_image">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>821</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="mb_show_kor">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>821</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>점프하다</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>mb_show_adj</zorder>
   <zorder>mb_voca_adj</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
